package.json

```diff
@@ -4,13 +4,9 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "docz dev",
-    "build": "docz build",
-    "test": "docz build",
-    "react:start": "react-scripts start",
-    "react:build": "react-scripts build",
-    "react:test": "react-scripts test",
-    "react:eject": "react-scripts eject"
+    "start": "lerna run build && docz dev",
```

Comment:
Mostly for my own edification: Under what circumstances do you need the  `--stream` flag here? I just noticed in #45 that we're using it, but here we're not.

Reply:
I believe we may want it here as well. `--stream` allows us to see interwoven logs for the concurrent scripts.